lotus-common/src/main/java/cn/metsea/lotus/common/enums/JobInstanceStatusEnum.java

```diff
@@ -0,0 +1,30 @@
+package cn.metsea.lotus.common.enums;
+
+import com.baomidou.mybatisplus.annotation.EnumValue;
+import com.fasterxml.jackson.annotation.JsonValue;
+import lombok.Getter;
+
+@Getter
+public enum JobInstanceStatusEnum {
+    WAITING_START(10, "waitingStart", false),
+    STARTING(20, "starting", false),
+    RUNNING(30, "running", false),
+    START_FAILED(40, "startFailed", true),
+    RUN_FAILED(50, "runFailed", true),
+    STOPPED(60, "stopped", true),
+    SUCCESS(70, "success", true),
+    UNKNOWN(-1, "unknown", false);
+
+    @EnumValue
+    private Integer code;
+    @JsonValue
+    private String desc;
+
+    private boolean FinalState;
```

Comment:
```suggestion
    private boolean FinalState;
```
与上面的 Status 不一致

FinalState 与 isFinal 语义歧义问题